tweener/tweener.lua

```diff
@@ -38,28 +37,35 @@ function M.tween(easing_function, from, to, time, callback, update_delta_time)
 
 	local time_elapsed = 0
 	local latest_time = socket.gettime()
+	local timer_table = { id = nil, is_paused = false }
+
+	timer_table.id = timer.delay(update_delta_time, true, function(_, handle, dt)
 
-	local timer_id = timer.delay(update_delta_time, true, function(_, handle, dt)
 		if time <= 0 then
-			timer.cancel(handle)
+			timer.cancel(timer_table.id)
 			callback(to, true)
 			return
 		end
 
 		local current_time = socket.gettime()
-		time_elapsed = time_elapsed + (current_time - latest_time)
+
+        if timer_table.is_paused == true then
+            time_elapsed = time_elapsed + (current_time - latest_time)
+        end
 		latest_time = current_time
 
 		if time_elapsed >= time then
-			timer.cancel(handle)
+			timer.cancel(timer_table.id)
```

Comment:
Not the timer can be finished, but the variable will never be deleted
so at tween which timer is gone, the function `tweener.exists` will return true, due the `id` is not removed in this case

Let's add `timer_table.id = nil` to fix this